kornia/testing/__init__.py

```diff
@@ -2,7 +2,10 @@
 The testing package contains testing-specific utilities.
 """
 from abc import ABC, abstractmethod
+from itertools import product
+from copy import deepcopy
 
+import pytest
```

Comment:
if we are making BaseTester dependent to `pytest` then we should move BaseTester outside the library core.

Reply:
@edgarriba How do you prefer this part of the refactor? I could move to `BaseTester` out of library core but since `BaseTester` is already used at quite a few places that would involve some refactor or I could move the fixture into the `CommonTests`?

Reply:
I think it's better to move the tester under ./test - I can do a quick patch this weekend.

Reply:
I tried to move it to ./test but it didn't really work due to importing issues, so I went for the second options instead.